tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/appsettings.json

```diff
@@ -212,23 +213,184 @@
     "ColumnNameModifiedDate": "Modified",
     "MatterConfigurationColumn": "ConfigurationValue",
     "SearchColumnsUIPickerForMatter": {
-      "Column1": "matterName",
-      "Column2": "matterClient",
-      "Column3": "matterClientId",
-      "Column4": "matterModifiedDate",
-      "Column5": "matterResponsibleAttorney",
-      "Column6": "matterSubAreaOfLaw",
-      "Column7": "matterCreatedDate"
+      "Column1": {
+            "displayInUI": true,
+            "field": "matterName",
+            "position":2
+        },
+        "Column10": {
+            "displayInUI": false,
+            "field": "matterClientUrl",
+            "position": -1
+        },
+        "Column11": {
+            "displayInUI": false,
+            "field": "matterPracticeGroup",
+            "position": -1
+        },
+        "Column12": {
+            "displayInUI": false,
+            "field": "matterAreaOfLaw",
+            "position": -1
+        },
+        "Column13": {
+            "displayInUI": false,
+            "field": "hideUpload",
+            "position": -1
+        },
+        "Column14": {
+            "displayInUI": false,
+            "field": "matterID",
+            "position": -1
+        },
+        "Column15": {
+            "displayInUI": false,
+            "field": "matterGuid",
+            "position": -1
+        },
+        "Column2": {
+            "displayInUI": true,
+            "field": "matterClient",
+            "position": 4
+        },
+        "Column3": {
+            "displayInUI": true,
+            "field": "matterClientId",
+            "position": 3
+        },
+        "Column4": {
+            "displayInUI": true,
+            "field": "matterModifiedDate",
+            "position": 5
+        },
+        "Column5": {
+            "displayInUI": true,
+            "field": "matterResponsibleAttorney",
+            "position":6 
+        },
+      "Column6": {
+        "displayInUI": true,
+        "field": "matterSubAreaOfLaw",
+        "position": 7
+      },
+      "Column7": {
+          "displayInUI": true,
+          "field": "matterCreatedDate",
+          "position": 8
+      },
+      "Column8": {
+          "displayInUI": false,
+          "field": "matterDescription",
+          "position": -1
+      },
+      "Column9": {
+          "displayInUI": false,
+          "field": "matterUrl",
+          "position": -1
+      }
     },
     "SearchColumnsUIPickerForDocument": {
-      "Column1": "documentName",
-      "Column2": "documentClient",
-      "Column3": "documentClientId",
-      "Column4": "documentModifiedDate",
-      "Column5": "documentOwner",
-      "Column6": "documentVersion",
-      "Column7": "documentCheckoutUser",
-      "Column8": "documentCreatedDate"
+      "DocColumn1": {
```

Comment:
Extended the **SearchColumnsUIPickerForDocument** json object to cater the needs of both project center and matter center when displaying the columns in the **document** search grid